### ISSUE TITLE
Messenger Incoming Calls

### ISSUE THREAD

**Issue by caingatcarl** [0x192/Universal-Android-Debloater#382]
**Your phone**: Samsung S22+

**Describe the issue**
I have an issue in the messenger app. I really don't know if it is related to debloating but incoming calls wont show up on the screen, even when I'm on the actual conversation of someone who is calling me. Sound or ringtone is nothing even in the notifications except when the call ends, then notification shows that I had a missed call.

**You have a solution?**
Tried all the solutions can be found on google related to that manner but the issue is still there.

**UAD log**
[UAD_20220906.log](https://github.com/0x192/universal-android-debloater/files/9571170/UAD_20220906.log)

**Comment by AnonymousWP** [comment]
Which packages did you remove?

**Comment by caingatcarl** [comment]
I don't know which one but i uninstalled 299. Should i upload the export file?

**Comment by AnonymousWP** [comment]
> I don't know which one but i uninstalled 299. Should i upload the export file?

Yes. We can see it in the log too, but that's not as efficient when reading.

**Comment by caingatcarl** [comment]
[uad_exported_selection.txt](https://github.com/0x192/universal-android-debloater/files/9579604/uad_exported_selection.txt)

**Comment by caingatcarl** [comment]
I'm referring to facebook messenger app btw.

**Comment by caingatcarl** [comment]
> > I don't know which one but i uninstalled 299. Should i upload the export file?
> 
> Yes. We can see it in the log too, but that's not as efficient when reading.

Hello sir how is it?

**Comment by AnonymousWP** [comment]
Well, start with restoring the Facebook-related applications and then try again.
